### PR TITLE
feat(aws): Support AWS Account Contacts

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: plugins/source/aws/codegen/recipes/account.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func AccountResources() []*Resource {
+	resources := []*Resource{
+		{
+			SubService:  "alternate_contacts",
+			Struct:      &types.AlternateContact{},
+			Description: "https://docs.aws.amazon.com/acm/latest/APIReference/API_CertificateDetail.html",
```
Comment (hermanschaaf) [inline plugins/source/aws/codegen/recipes/account.go:14]: This description doesn't look right

File: plugins/source/aws/codegen/recipes/account.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+					Type:     schema.TypeString,
+					Resolver: "client.ResolveAWSAccount",
+					Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+				},
+			},
+		},
+		{
+			SubService:  "contacts",
+			Struct:      &types.ContactInformation{},
+			Description: "https://docs.aws.amazon.com/acm/latest/APIReference/API_CertificateDetail.html",
```
Comment (hermanschaaf) [inline plugins/source/aws/codegen/recipes/account.go:28]: This one too

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
LGTM, just the descriptions that need to be updated I think

File: plugins/source/aws/codegen/recipes/account.go (hunk 1 above)
Comment (bbernays) [reply plugins/source/aws/codegen/recipes/account.go]: fixed

File: plugins/source/aws/codegen/recipes/account.go (hunk 2 above)
Comment (bbernays) [reply plugins/source/aws/codegen/recipes/account.go]: fixed

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**